manifests/mod/remoteip.pp

```diff
@@ -51,12 +51,12 @@
 #
 class apache::mod::remoteip (
   String                                                     $header                    = 'X-Forwarded-For',
-  Optional[Array[Variant[Stdlib::Host,Stdlib::IP::Address]]] $internal_proxy            = undef,
-  Optional[Array[Variant[Stdlib::Host,Stdlib::IP::Address]]] $proxy_ips                 = undef,
+  Optional[Array[Stdlib::IP::Address]]                       $internal_proxy            = undef,
+  Optional[Array[Stdlib::IP::Address]]                       $proxy_ips                 = undef,
```

Comment:
Wait, "The type Stdlib::Host includes Stdlib::IP::Address", was it not supposed to be the other way around?

```
  Optional[Array[Stdlib::Host]]                              $internal_proxy            = undef,
  Optional[Array[Stdlib::Host]]                              $proxy_ips                 = undef,
```

:thinking:

Reply:
Bleh, that's what you get for quickly doing the work.

Reply:
https://github.com/puppetlabs/puppetlabs-apache/pull/2458